From 2560a6b7b9adc5bd5dec0f87c2e3025ced8a2af2 Mon Sep 17 00:00:00 2001
From: Andrej Valek <andrej.valek@siemens.com>
Date: Thu, 18 Apr 2019 10:47:11 +0200
Subject: [PATCH] fs-utilh: add missing sys/stat include

fix error:
| error: passing argument 5 of 'chase_symlinks_and_stat' from incompatible pointer type [-Werror=incompatible-pointer-types]

Signed-off-by: Andrej Valek <andrej.valek@siemens.com>
---
 src/basic/fs-util.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/basic/fs-util.h b/src/basic/fs-util.h
index b965120..e2fdfff 100644
--- a/src/basic/fs-util.h
+++ b/src/basic/fs-util.h
@@ -8,6 +8,7 @@
 #include <stdint.h>
 #include <sys/inotify.h>
 #include <sys/types.h>
+#include <sys/stat.h>
 #include <unistd.h>
 
 #include "errno-util.h"
-- 
2.11.0

